dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -266,6 +298,191 @@ fn first_page<'a, T>(limit: NonZeroU32) -> DataPageParams<'a, T> {
     }
 }
 
+// Disks
+
+/// Run `omdb db disk list`.
+async fn cmd_db_disk_list(
+    datastore: &DataStore,
+    limit: NonZeroU32,
+) -> Result<(), anyhow::Error> {
+    #[derive(Tabled)]
+    #[tabled(rename_all = "SCREAMING_SNAKE_CASE")]
+    struct DiskRow {
+        name: String,
+        id: String,
+        size: String,
+        state: String,
+        attached_to: String,
+    }
+
+    let mut rows = Vec::with_capacity(2);
+    let ctx = || "listing disks".to_string();
+
+    use db::schema::disk::dsl;
+    let disks = dsl::disk
+        .filter(dsl::time_deleted.is_null())
+        .limit(i64::from(u32::from(limit)))
+        .select(Disk::as_select())
+        .load_async(datastore.pool_for_tests().await?)
+        .await
+        .context("loading disks")?;
+
+    check_limit(&disks, limit, ctx);
+
+    rows.extend(disks.into_iter().map(|disk| DiskRow {
```

Comment:
nit: you could just `let rows = disks.into_iter()...` at this point, instead of creating a `mut rows` above and extending it here.

Reply:
Done